src/appleseed.bench/mainwindow/rendering/renderwidget.cpp

```diff
@@ -176,7 +176,9 @@ namespace
 void RenderWidget::highlight_tile(
     const Frame&    frame,
     const size_t    tile_x,
-    const size_t    tile_y)
+    const size_t    tile_y,
+    const size_t    thread_index,
+    const size_t    nb_threads)
```

Comment:
As far as I understand, only the `RenderWidget` needs `thread_index` and `nb_threads`. Have you tried to give those informations to the `RenderWidget` without affecting all tile callbacks (such as `stdout`) ?

Reply:
We use the "count" terminology throughout the code base, i.e. `thread_count` rather than `nb_threads`.

Reply:
`thread_index` can't be passed just to `RenderWidget` since by definition it varies per thread, and there's only one `RenderWidget`. It must be passed to `highlight_tile()`.

It's a different matter for `thread_count`, but if we're passing one, we can as well pass both.